src/main/java/org/jabref/gui/texparser/ParseTexResultView.java

```diff
@@ -0,0 +1,72 @@
+package org.jabref.gui.texparser;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.ListView;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import javafx.scene.text.Text;
+
+import org.jabref.gui.util.BaseDialog;
+import org.jabref.gui.util.ViewModelListCellFactory;
+import org.jabref.logic.l10n.Localization;
+import org.jabref.model.strings.LatexToUnicodeAdapter;
+import org.jabref.model.texparser.Citation;
+import org.jabref.model.texparser.TexParserResult;
+
+import com.airhacks.afterburner.views.ViewLoader;
+import org.fxmisc.easybind.EasyBind;
+
+public class ParseTexResultView extends BaseDialog<Void> {
+
+    private final TexParserResult texParserResult;
+    @FXML private ListView<ReferenceWrapper> referenceListView;
+    @FXML private ListView<Citation> citationListView;
+    private ParseTexResultViewModel viewModel;
+
+    public ParseTexResultView(TexParserResult texParserResult) {
+        this.texParserResult = texParserResult;
+
+        this.setTitle(Localization.lang("Bibliographic entries search"));
+
+        ViewLoader.view(this)
+                  .load()
+                  .setAsDialogPane(this);
+    }
+
+    @FXML
+    private void initialize() {
+        viewModel = new ParseTexResultViewModel(texParserResult);
+
+        referenceListView.itemsProperty().setValue(viewModel.getReferenceList());
+        EasyBind.subscribe(referenceListView.getSelectionModel().selectedItemProperty(), reference -> {
+            if (reference != null) {
+                viewModel.getCitationList().setAll(reference.getCitationList());
```

Comment:
The view should never update the view model. What would work is if you introduce a new `activeReferenceChanged(reference)` method in the view model, which then updates the list of citations.